css/styles.css

```diff
@@ -6,10 +6,14 @@ body {
     font-family: 'Press Start 2P', cursive;
 }
 
-.container {
+#container {
     width: 1000px;
     height: 800px;
     margin: 0 auto;
     border: 1px solid black;
 
+}
+
```

Comment:
also add a display: block; to the main within ie9 to fix the border and positioning issues.

Reply:
🎉